homeassistant/components/homematicip_cloud.py

```diff
@@ -96,6 +97,7 @@ class HomematicipConnector:
     def __init__(self, hass, config, websession):
         """Initialize HomematicIP cloud connection."""
         from homematicip.async.home import AsyncHome
```

Comment:
In Python 3.7, async becomes a protected keyword. This package needs to be renamed.

Reply:
Ok. Will do that. I will make a separate PR for this ?

Reply:
Ok